internal_transfers/actions/tests/database.spec.ts

```diff
@@ -82,4 +89,101 @@ describe("test database insertion methods", () => {
     results = await db.query(sql`SELECT * from internalized_imbalances;`);
     expect(results).toStrictEqual(expectedResults);
   });
+  test("insertSimulations(datum) & idempotent", async () => {
+    const blockNumber = 1;
+    const largeBigInt = 99999999999999999999999999999999999999999999999999n;
+    const exampleLog = {
+      address: zeroHex,
```

Comment:
Nit: this should be `zeroAddress`. (n practice it doesn't really matter what this is because it's copied over verbatim.)

Reply:
fixed.